dask_sql.Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN gpuci_mamba_retry env create -n dask_sql --file /dask_sql_environment.yaml
 RUN gpuci_mamba_retry install -y -n dask_sql -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
     cudatoolkit=$CUDA_VER \
     cudf=$RAPIDS_VER \
+    cuml=$RAPIDS_VER \
+    xgboost \
```

Comment:
Should we pin the xgboost version with a variable like `XGBOOST_VER`?

Reply:
I dont think we should. We should have it working with the latest version .  As the release cycles of both `xgboost` and `rapids` vary  it will add a lot of overhead to keep this updated.

Reply:
That's fair - ideally we can always restrict this later on if we run into any breakage

Reply:
Agreed.